sysboundary/antisymmetric.cpp

```diff
@@ -121,6 +125,46 @@ namespace SBC {
             mpiGrid[cells[c]]->sysBoundaryFlag = flag;
          }
       }
+
+      const auto inv2powMaxRefLvl = pow(2,-P::amrMaxSpatialRefLevel);
+      
+      // Assign boundary flags to local fsgrid cells
+      const std::array<int, 3> gridDims(technicalGrid.getLocalSize());  
+      for (int k=0; k<gridDims[2]; k++) {
+         for (int j=0; j<gridDims[1]; j++) {
+            for (int i=0; i<gridDims[0]; i++) {
+               const auto& coords = technicalGrid.getPhysicalCoords(i,j,k);
+
+               // Shift to the center of the fsgrid cell
+               auto cellCenterCoords = coords;
+               cellCenterCoords[0] += 0.5 * P::dx_ini * inv2powMaxRefLvl;
```

Comment:
This code is quite complex to get the fsgrid dx, it also makes the assumption fsgriddx is the same as finest level dccrg grid one (though we do that in many places so ok). Now fsgrid leavs the DX DY DZ as public variables, perhaps just use those? I guess somebody may want to later add a get function for those

Reply:
Oh, okay, I did not realize they are public. But indeed we do make the assumption that fsgrid has the same resolution as the finest level dccrg in a lot of places right now.